tests/phpunit/tests/block-bindings/postMetaSource.php

```diff
@@ -38,6 +38,15 @@ public static function wpTearDownAfterClass() {
 		$GLOBALS['wp_meta_keys'] = self::$wp_meta_keys_saved;
 	}
 
+	/**
```

Comment:
This helper can be inlined in the test to make it easier to follow.

Reply:
Done, but auto-formatting is against it. 😅